ext/opcache/jit/zend_jit_helpers.c

```diff
@@ -3030,7 +3030,8 @@ static zend_result ZEND_FASTCALL zval_jit_update_constant_ex(zval *p, zend_class
 		} else {
 			zval tmp;
 
-			if (UNEXPECTED(zend_ast_evaluate(&tmp, ast, scope) != SUCCESS)) {
+			zend_ast_evaluate_ctx ctx = {0};
+			if (UNEXPECTED(zend_ast_evaluate(&tmp, ast, scope, &ctx) != SUCCESS)) {
```

Comment:
I think this should be (although I don't understand why CI passed):
```suggestion
			if (UNEXPECTED(zend_ast_evaluate_ex(&tmp, ast, scope, &ctx) != SUCCESS)) {
```

Reply:
Ah, good catch! This code is excluded via preprocessor.

Reply:
Hmm...

```c
#if (PHP_VERSION_ID <= 80100) && (SIZEOF_SIZE_T == 4)
```

From the commit message:

> Keep the same JIT code for PHP-8.0 and master (workaround against PHP-8.0 ABI freeze)

We can probably remove this code now.

Reply:
Ah, good find. I suggest to have a separate commit (or PR) for this.